### DISCUSSION TITLE
Samsung Smart Remote

### DISCUSSION THREAD

**Discussion by OptimusGREEN** [PRProd/HA-Firemote#561] [New Device Family Support Request]
Hi, love the project. I'm just on to request support for the samsung tv smart remote. that would be a great addition.

thanks again

**Comment by PRProd** [comment]
Greetings @OptimusGREEN!

You're in luck!  There's a special place in the discussions for the community to vote on what gets added next!  Take a look at this discussion thread, and place your vote there!

[Poll: Vote for what gets added next!](https://github.com/PRProd/HA-Firemote/discussions/41)

Thanks for stopping by and saying hello!

-Doug

**Reply by OptimusGREEN** [reply]
great. thanks for the info.
